brave/brave-core
# Do not load extensions in non-incognito guest profile

Two guest profiles are created. First, a regular on-the-record Profile is created, and then the off-the-record version of that Profile is created and returned to be used by a window. However, we were previously allowing extensions to be loaded in both these profiles, resulting in background scripts loading twice and receiving the same tab events.

Fix https://github.com/brave/brave-browser/issues/5564

## Test plan
1. On master, follow the plan in https://github.com/brave/brave-browser/issues/5564
2. Switch to this branch in brave-core
3. `npm run sync` (no params) to reset the previously-patched file
4. Follow the plan again in https://github.com/brave/brave-browser/issues/5564 and verify fixed
5. Verify shields still works in guest and tor profiles


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
